internal/network/command_transit_gateway_attachment.go

```diff
@@ -0,0 +1,134 @@
+package network
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	networkingv1 "github.com/confluentinc/ccloud-sdk-go-v2/networking/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type transitGatewayAttachmentCommand struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+type transitGatewayAttachmentHumanOut struct {
+	Id                         string `human:"ID"`
+	Name                       string `human:"Name"`
+	NetworkId                  string `human:"Network ID"`
+	RamShareArn                string `human:"RAM Share ARN"`
+	TransitGatewayId           string `human:"Transit Gateway ID"`
+	Routes                     string `human:"Routes"`
+	TransitGatewayAttachmentId string `human:"Transit Gateway Attachment ID,omitempty"`
+	Phase                      string `human:"Phase"`
+}
+
+type transitGatewayAttachmentSerializedOut struct {
+	Id                         string   `serialized:"id"`
+	Name                       string   `serialized:"name"`
+	NetworkId                  string   `serialized:"network_id"`
+	RamShareArn                string   `serialized:"ram_share_arn"`
+	TransitGatewayId           string   `serialized:"transit_gateway_id"`
+	Routes                     []string `serialized:"routes"`
+	TransitGatewayAttachmentId string   `serialized:"transit_gateway_attachment_id,omitempty"`
+	Phase                      string   `serialized:"phase"`
+}
+
+func newTransitGatewayAttachmentCommand(prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "transit-gateway-attachment",
+		Aliases: []string{"tgw-attachment"},
```

Comment:
Not much of an alias... this is still longer than most of our existing commands. What about `confluent network tga`?

Reply:
thats fine

Reply:
Updated to `tgwa`